vertx-db2-client/src/test/java/io/vertx/db2client/tck/DB2TransactionTest.java

```diff
@@ -74,4 +71,19 @@ public void testDelayedCommit(TestContext ctx) {
     super.testDelayedCommit(ctx);
   }
 
+  @Override
```

Comment:
@aguibert I failed to make these tests pass, can you help with this? I tried to set transaction characteristic but I got this
```
>>> BEGIN DB2TransactionTest.testStartReadOnlyTransaction
[2020-05-18 23:53:30] 详细 io.vertx.db2client.impl.codec.DB2Encoder                >>> ENCODE SimpleQueryCommandCodec@2e09e4e1 sql=SET TRANSACTION READ ONLY, autoCommit=false, section=null
[2020-05-18 23:53:30] 详细 io.vertx.db2client.impl.codec.DB2Decoder                <<< DECODE SimpleQueryCommandCodec@2e09e4e1 sql=SET TRANSACTION READ ONLY, autoCommit=false, section=io.vertx.db2client.impl.drda.Section$ImmediateSection@7b071208{packageName=SYSLH200, sectionNumber=385, cursorName=SQL_CURLH200C} (123 bytes)

io.vertx.db2client.DB2Exception: The SQL syntax provided was invalid
```